core/types/block.libevm_test.go

```diff
@@ -29,13 +29,16 @@ import (
 	. "github.com/ava-labs/libevm/core/types"
 	"github.com/ava-labs/libevm/crypto"
 	"github.com/ava-labs/libevm/libevm/ethtest"
+	"github.com/ava-labs/libevm/libevm/pseudo"
 	"github.com/ava-labs/libevm/rlp"
 )
 
 type stubHeaderHooks struct {
 	suffix                                   []byte
 	gotRawJSONToUnmarshal, gotRawRLPToDecode []byte
 	setHeaderToOnUnmarshalOrDecode           Header
+	accessor                                 pseudo.Accessor[*Header, *stubHeaderHooks]
+	copied                                   *stubHeaderHooks
```

Comment:
```suggestion
	accessor                                 pseudo.Accessor[*Header, *stubHeaderHooks] // from [RegisterExtras]
	toCopy                                   *stubHeaderHooks
```

`copied` implies that it's the "got" / "actual" of the test whereas `toCopy` implies that it's an action for the test double.